### PR TITLE
Improve User interface.

### PR DESCRIPTION
**Pull request by sparshg**
@chimosky Please review.

- Add ripple effect to background image, add noise, make darker.
- Center the text elements at top.
- Change colors of text elements to provide better readability.
- Draw circle around falling numbers so that thy can be clicked easily.

### CODE REVIEW

**Comment by chimosky** [comment]
Tested 3fca022, some comments;

- This is one of those cases where splitting some of the changes into various commits would be more helpful for review.
- Could you add some more space to the circles as they're mostly joined together and it doesn't look really good.

**Comment by sparshg** [comment]
Splitted commits, spaced out numbers and also fixed the overlapping numbers problem.

**Review by chimosky — COMMENTED** [review]
Why do you think the ripple effect helps improve the UI?

File: juego.py [hunk 1]
```diff
@@ -67,24 +65,14 @@ def __init__(self, level, fuente):
...
-            count_x = 0
-            while count_x < len(list_x):
-                rand_coord_x = random.choice(list_x)
-                count_x += 1
-                if rand_coord_x not in no_repeat_check_x:
-                    no_repeat_check_x.append(rand_coord_x)
-                    return rand_coord_x
-            return int(sx(1000))
+            rand_coord_x = random.choice(list_x)
+            list_x.remove(rand_coord_x)
```
Comment (chimosky) [inline juego.py:69]: Why remove an item you just created? @sparshg

**Comment by sparshg** [comment]
The character in the background was repeatedly tiled, the ripple effect gave a bit of random variations to the expressions of characters which seemed interesting.

**Comment by chimosky** [comment]
Tested 30609c8, activity fails to start with this in the logs;

```
Traceback (most recent call last):
  File "/home/ibiam/Activities/jamath-activity/juego.py", line 537, in run
    self.play(level)
  File "/home/ibiam/Activities/jamath-activity/juego.py", line 342, in play
    nueva_expresion = expresion(level, self.fuente_60)
  File "/home/ibiam/Activities/jamath-activity/juego.py", line 78, in __init__
    self.correct_number = number(rand_generator_x(), rand_generator_y(),
  File "/home/ibiam/Activities/jamath-activity/juego.py", line 74, in rand_generator_y
    list_x.remove(rand_coord_y)
ValueError: list.remove(x): x not in list
1661342719.647600 ERROR root: Event queue full!
1661342719.654984 ERROR root: Event queue full!
1661342719.662592 ERROR root: Event queue full!
1661342719.678005 ERROR root: Event queue full!
1661342719.685460 ERROR root: Event queue full!
1661342719.737421 ERROR root: Event queue full!
1661342719.745524 ERROR root: Event queue full!
Traceback (most recent call last):
  File "/home/ibiam/Activities/jamath-activity/sugargame/event.py", line 115, in _visibility_cb
    self.update_display()
  File "/home/ibiam/Activities/jamath-activity/sugargame/event.py", line 98, in update_display
    pygame.event.post(pygame.event.Event(pygame.VIDEOEXPOSE))
pygame.error: Event queue full
```

Started the activity again and it failed to start with just this in the logs;
```
Traceback (most recent call last):
  File "/home/ibiam/Activities/jamath-activity/juego.py", line 537, in run
    self.play(level)
  File "/home/ibiam/Activities/jamath-activity/juego.py", line 342, in play
    nueva_expresion = expresion(level, self.fuente_60)
  File "/home/ibiam/Activities/jamath-activity/juego.py", line 78, in __init__
    self.correct_number = number(rand_generator_x(), rand_generator_y(),
  File "/home/ibiam/Activities/jamath-activity/juego.py", line 74, in rand_generator_y
    list_x.remove(rand_coord_y)
ValueError: list.remove(x): x not in list
```

It also seems the event queue isn't been handled properly.

**Comment by chimosky** [comment]
> The character in the background was repeatedly tiled, the ripple effect gave a bit of random variations to the expressions of characters which seemed interesting.

Yes it does seem interesting, I'm asking how that helps improve the UI.

**Comment by sparshg** [comment]
> Yes it does seem interesting, I'm asking how that helps improve the UI.

I included some changes like this that focus on the visuals and not just UI, for example changing the color of numbers and aligning text elements to center. The PR was intended to improve visuals along with UI.

**Comment by chimosky** [comment]
> I included some changes like this that focus on the visuals and not just UI, for example changing the color of numbers and aligning text elements to center. The PR was intended to improve visuals along with UI.

The question was about that particular change and I wasn't opposed to the change, just wanted to understand your thought process, the target for the activity is kids and a rippled background might seem appealing to a kid, that was the sort of response I was looking for.

**Comment by sparshg** [comment]
Oh misunderstood the question a bit 😅. I thought you meant that this PR wasn't for visual effects.

**Comment by chimosky** [comment]
Could you respond to [this](https://github.com/sugarlabs/jamath-activity/pull/25#discussion_r953713794) question?

File: juego.py (hunk 1 above)
Comment (sparshg) [reply juego.py:69]: So basically we choose a random position from the list of predefined positions, and return that. Also to make sure that no overlapping with other numbers occur, we remove the selected position from the list.

**Comment by chimosky** [comment]
Reviewed, thanks for the change!

**Pull request merged**